components/host-dashboard/AddFundsModal.js

```diff
@@ -500,6 +505,14 @@ const AddFundsModal = ({ collective, ...props }) => {
                   >
                     {({ field }) => <StyledInput data-cy="add-funds-description" {...field} />}
                   </StyledInputFormikField>
+                  <StyledInputFormikField
+                    name="memo"
+                    htmlFor="addFunds-memo"
+                    label={<FormattedMessage defaultMessage="Memo (Optional)" />}
```

Comment:
Using required = {false} already adds a optional hint.
```suggestion
                    required={false}
                    label={<FormattedMessage defaultMessage="Memo" />}
```

![image](https://user-images.githubusercontent.com/5448927/196681396-2cb8d2d3-1e1d-4261-83de-2e8b70e3cb20.png)

Reply:
Thanks. Missed this. 😉